### PR TITLE
Gdpr

### LINKED ISSUE #1138: GDPR compliance

### PR DESCRIPTION
**Pull request by brunobesson**
Fixes #1138

### CODE REVIEW

File: src/components/gdpr/GdprBanner.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  top: 0;
+  padding: 10px;
+
+  margin: 0 auto;
+  flex-direction: column;
+  display: flex;
+  background-color: white;
+  box-shadow: none;
+  transform: translateY(-100%);
+  transition: all 0.25s ease-in-out;
```
Comment (brunobesson) [inline src/components/gdpr/GdprBanner.vue:87]:
```suggestion
  transition: transform 0.25s ease-in-out;
```

**Comment by cbeauchesne** [comment]
@brunobesson this change was something you wanted you did on purpose ? 🤔 

https://github.com/c2corg/c2c_ui/pull/1646/files#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79R35

**Comment by brunobesson** [comment]
Yes, indeed. Why?
It is enabled later on in `gdpr.js` if you allowed analytics.

**Comment by cbeauchesne** [comment]
We have no data on google analytics since the release, we may have a bug somewhere

**Pull request merged**